File: base/src/main/java/de/hipp/pnp/converter/Attribute5eConverter.java
package de.hipp.pnp.converter;

import de.hipp.pnp.Attribute5e;

import javax.persistence.Converter;

@Converter(autoApply = true)
public class Attribute5eConverter extends BaseConverter<Attribute5e> {

    public Attribute5eConverter() {
        this.type = new Attribute5e();
    }
}

File: src/main/kotlin/de/hipp/pnp/base/converter/persistent/Attribute5eConverter.kt
package de.hipp.pnp.base.converter.persistent

import de.hipp.pnp.base.fivee.Attribute5e
import jakarta.persistence.Converter
import org.springframework.stereotype.Component

@Converter(autoApply = true)
@Component
class Attribute5eConverter : BaseConverter<Attribute5e?>() {
    init {
        this.type = Attribute5e()
    }
}
